lib/rules/no-useless-rename.js

```diff
@@ -109,7 +115,7 @@ module.exports = {
             }
 
             if (node.imported.name === node.local.name &&
-                    node.imported !== node.local) {
+                    !lodash.isEqual(node.imported, node.local)) {
```

Comment:
Can't you just check that `range[0]` isn't the same? We really only care about the location, right?